.github/workflows/bazel-build.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
haven't you already taken care of this in https://github.com/p4lang/PI/pull/620?
could you rebase this PR?

Reply:
Yes, I went ahead and threw it in this PR as well, to see if CI passed or not.  I will rebase.